assets/css/styles.css

```diff
@@ -4,29 +4,159 @@
   margin: 0;
   padding: 0;
   font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
 }
 
-.topHeader {
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  padding: 3% 6.3% 0 6.2%;
-  box-sizing: border-box;
+html {
+  scroll-behavior: smooth;
+}
+
+header {
+  background-color: white;
 }
 
 .logo {
+  width: 50%;
+  height: 100%;
   font-style: normal;
   font-weight: 700;
-  font-size: 1.125rem;
-  line-height: 1.25rem;
-  color: #6070ff;
+  font-size: 18px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  background-color: #fff;
+  padding: 2px;
+}
+
+a {
+  color: #344563;
+  text-decoration: none;
 }
 
 .logo a {
+  margin-left: 24px;
+  background-color: #fff;
+  color: #5464f8;
   text-decoration: none;
-  color: #6070ff;
 }
 
+li {
+  list-style: none;
+}
+
+.nav-bar {
+  min-height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #5464f8;
+}
+
+.nav-bar .bar {
+  background-color: #5464f8;
+}
+
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 60px;
+  margin-right: 50px;
+}
+
+.nav-branding {
+  font-style: normal;
+  font-weight: 700;
```

Comment:
* Kindly fix the difference between your design and the given Figma design 
* as in your design the links spacing is not the same as the given template
* The font sizes should match that of the Figma 
* The background color should cover the whole view on mobile as shown in the Figma and it should be transparent but blurred  
* ## YOURS 
![image](https://user-images.githubusercontent.com/52497246/232864533-6c482b89-bcb0-4b9c-8fc0-9359035aea7b.png)
## FIGMA 
![image](https://user-images.githubusercontent.com/52497246/232864625-69a10dc5-e33f-4b28-8e8f-5f867855dd3c.png)

Reply:
![fixed](https://user-images.githubusercontent.com/86327867/232950446-3458f3bf-0285-44cb-8819-3ed0ec2d03d1.jpg)


I hope that this fix can cover the suggestion.